src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -3,5 +3,5 @@
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws RegisterException;
```

Comment:
No need to add unchecked exception to method signature (throws)
```suggestion
    User register(User user);
```